lib/git-lfs-s3/application.rb

```diff
@@ -47,6 +49,151 @@ def protected!
       "Git LFS S3 is online."
     end
 
+    def valid_object?(object)
+      begin
+        valid = object[:size] >= 0
+      rescue
+        valid = false
+      end
+      begin        
+        if valid
+          oid = object[:oid].hex
+          valid = oid.size == 32 && object[:oid].size == 64
+        end
+      rescue
+        valid = false
+      end
+      valid
+    end
+
+    def object_download(authenticated, object, object_json)
+      # Format a single download object.
+      oid = object_json[:oid]
+      size = object_json[:size]
+      {
+        'oid' => oid,
+        'size' => size,
+        'authenticated' => authenticated,
+        'actions' => {
+          'download' => {
+            'href' => object.presigned_url(:get, :expires_in => 86400)
+          }
+        },
+        'expires_at' => DateTime.now.next_day.to_time.utc.iso8601
```

Comment:
Yeah I didn't know about trailing commas or lining up hash rockets. I'll do that now.